packages/components/src/inputs/SelectInput/SelectInput.tsx

```diff
@@ -23,10 +23,10 @@ export type Option = {
 type OwnProps = {
   options: Option[];
   disableVariant?: 'text' | 'default';
-  variant?: 'standard' | 'outlined';
+  variant?: 'standard' | 'outlined' | 'filled';
 };
 
-type SelectInputProps = OwnProps & ComponentPropsWithoutRef<typeof TextInput>;
+type SelectInputProps = OwnProps & Omit<ComponentPropsWithoutRef<typeof TextInput>, 'variant'>;
```

Comment:
MUI also has exactly same variants. Do we still need to omit them from TextInput props?